browser/src/map/handler/Map.KeyboardShortcuts.ts

```diff
@@ -88,11 +94,14 @@ class KeyboardShortcuts {
         const viewType = this.map.isEditMode() ? ViewType.Edit : ViewType.ReadOnly;
 
         const shortcuts = descriptors.filter((descriptor: ShortcutDescriptor) => {
+            const keyMatches = descriptor.key === key;
+            const keyCodeMatches = Array.isArray(descriptor.keyCode) ? descriptor.keyCode.includes(keyCode) : descriptor.keyCode === keyCode;
```

Comment:
keyCode cannot be array, not needed?

Reply:
keyCode must be an array to get some keys correctly - such as how any number on the keypad is treated as KP_9 rather than 9, say. Previously I was defining the keyCodes as an object as they were in Map.Keyboard.js, so it was obviously needed due to the typing system, so I added it

At present it isn't needed and I guess I could drop it and someone could extend it later if they want it again